clean.sh

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env bash
+
+usage()
+{
+    echo "Usage: clean [options]"
+    echo "Cleans the local dev environment."
+    echo
+    echo "  -b     Delete bin directory"
+    echo "  -p     Delete packages directory and NuGet package caches"
+    echo "  -a     Remove all untracked files in the working tree and delete NuGet package caches"
```

Comment:
Mariana, I agree with your suggestion for the documentation.

Wes, I'll add a "-w" option which cleans the entire working tree but doesn't touch the package cache.  I don't like "-s" as much because this is different than the "/s" option in the TFS clean.cmd script, which doesn't clean the binaries.

I like using "-a" for all, but I'll add "--all" as another option for it.

I'm thinking the options will be:
```
    echo "  -b         Delete bin directory"
    echo "  -p         Delete packages directory"
    echo "  -w         Remove all untracked files in the working tree, including the bin and packages directories"
    echo "  -c         Delete the user-local NuGet package caches"
    echo "  -a, --all  Clean all of the above"
```